src/apps/Makefile

```diff
@@ -0,0 +1,28 @@
+CC = clang
+
+objectify = $(subst .c,.o, $1)
+
+SRC = parser.c
+OBJS = $(call objectify,$(SRC))
+TARGETS = parser
+SANS = -fsanitize=leak,address
+
+INC=-I../../include -I/usr/include/json-c -I/usr/local/include/botan-2/
+
+# Ideally this should be removed at some point
+INC+=-I../../src/lib
+
+CFLAGS=$(SANS) $(INC) -g -O0
+LDFLAGS=
+
+%.o: %.c
+	$(CC) -c -o $@ $< $(CFLAGS)
+
+parser: $(OBJS)
+	$(CC) -o $@ $^ $(CFLAGS) -L../librepgp/.libs -lrepgp -L../lib/.libs -lrnp -L../librekey/.libs -lrekey
```

Comment:
I think that's using `.libs` as hardcoded path is very-very-very bad way, am I wrong?